packages/flutter/test/widgets/scrollable_semantics_test.dart

```diff
@@ -37,10 +37,12 @@ void main() {
 
     await flingDown(tester);
     expect(semantics, includesNodeWith(actions: <SemanticsAction>[SemanticsAction.scrollUp, SemanticsAction.scrollDown]));
+
+    semantics.dispose();
```

Comment:
What do you think of adding an `assert` in the constructor of `SemanticsTester` making sure that the semantics is off, and throwing something along the lines of "Semantics is already turn on. It is possible that a previous unit test failed to call `semantics.dispose` before it finished".

Also, maybe these should be moved into `setUp` so they are called regardless of whether the test succeeds or fails?

Reply:
I wish we could turn on semantics in setUp and turn it off again in tearDown. The former doesn't work though because we need an instance of WidgetTester to do that. Nevertheless, I refactored the code a little to do at least the dispose in tearDown. 

Adding the assert should probably be done in a separate PR as it affects many tests. Also not sure if we really want it. In theory you want to be able to test that everything works even if you have multiple semantics listeners (not that we currently do that anywhere).

Reply:
I see. nvm then.